allocate.go

```diff
@@ -200,6 +209,17 @@ func (a *ExecAllocator) Allocate(ctx context.Context, opts ...BrowserOption) (*B
 		return nil, err
 	}
 
+	// If requested, copy all terminal output from the browser
+	// instance (stdout and stderr) to given io.Writer. Note that
+	// this is different from the output written to the browser console.
+	if a.browserOutputWriter != nil {
+		go io.Copy(a.browserOutputWriter, stdout)
+		go io.Copy(a.browserOutputWriter, stderr)
```

Comment:
hm, this is bound to be racy. most writers don't accept concurrent writes.

I'd instead use cmd.CombinedPipe, which already takes care of this for you. Our code that picks up the websocket port shouldn't be affected.

Reply:
I don't see any cmd.CombinedPipe() - maybe I'm missing something? There's [cmd.CombinedOutput](https://golang.org/pkg/os/exec/#Cmd.CombinedOutput), but I don't think that's what we want given that it blocks until the command is complete.

Reply:
Using a multireader isn't what we want either, as it would return all of stdout before any stderr (or vice versa). I agree with your concern about races here though.

Reply:
Ah, my bad. It uses Stdout and Stderr directly, not the pipes, so perhaps do the same?

Honestly, we don't care so much about stdout and stderr being in order, since 99% of the time the user will only care about either of those. But two goroutines copying into the writer is definitely buggy.